openedx/core/djangoapps/catalog/utils.py

```diff
@@ -162,10 +162,10 @@ def munge_catalog_program(catalog_program):
             } for course in catalog_program['courses']
         ],
         'banner_image_urls': {
-            'w1440h480': catalog_program['banner_image']['large']['url'],
-            'w726h242': catalog_program['banner_image']['medium']['url'],
-            'w435h145': catalog_program['banner_image']['small']['url'],
-            'w348h116': catalog_program['banner_image']['x-small']['url'],
+            'w1440h480': catalog_program.get('banner_image', {}).get('large', {}).get('url'),
```

Comment:
I had programs locally and there are programs on staging that didn't have a banner image (though I think some might have had size missing as well).
I was just being extra safe here, but I could remove some of the gets that may not be necessary